resolwe_bio/processes/WGBS/walt.yml

```diff
@@ -45,19 +45,26 @@
         Sets the number of reads to mapping in each loop. Larger number results in program taking
         more memory. This is especially evident in paired-end mapping.
   output:
-    - name: mr
+    - name: bam
       label: Alignment file
```

Comment:
Update the labels for the `bam` and `mr` so that it will be possible to differentiate between the the output fields in the UI. One option is to use `Alignment file (BAM)` and `Alignment file (MR)`.